utils/getSupabaseClient.ts

```diff
@@ -3,5 +3,6 @@ import { createClient } from "@supabase/supabase-js";
 const supabaseUrl = process.env.NEXT_PUBLIC_DB_SERVER;
 const supabaseKey = process.env.NEXT_PUBLIC_SUPABASE_KEY;
 export const getSupabaseClient = () => {
+  // might not want to do this - it might call createClient way too often
```

Comment:
I think it's necessary, since there's no global state during build. Each page is a new build essentially, and therefore we need to reinstantiate every time.